Firmware/Marlin_main.cpp

```diff
@@ -7238,6 +7238,87 @@ SERIAL_PROTOCOLPGM("\n\n");
 
     }break;
     #endif // FWRETRACT
+    /*!
```

Comment:
It's nice that you documented the gcode. This should be standard in Prusa-Firmware as it keeps everything as tidy as possible. If I'm not mistaken, there's a section that is only for titles and links to all gcodes. It should be added there as well. https://github.com/prusa3d/Prusa-Firmware/blob/43c22e7fe3f0673a93defc2a6a3c25161400be23/Firmware/Marlin_main.cpp#L3511
@3d-gussner Is the one that wrote the gcode documentation in the firmware. He should be able to tell you more about this and make sure that it is documented properly everywhere

Reply:
Whoops.  I had that in my test project, but I copied the changes into a fresh fork to keep things clean.  I will add it back.  Here is what I had:

```
//!@n M214 - Set Arc Parameters (Use M500 to store in eeprom) N<MM_PER_ARC_SEGMENT> S<MIN_MM_PER_ARC_SEGMENT> R<MIN_ARC_SEGMENTS> F<ARC_SEGMENTS_PER_SEC>
```
Good catch.  @3d-gussner, what do you think?